packages/server/graphql/types/PokerTemplate.ts

```diff
@@ -17,9 +17,9 @@ const PokerTemplate = new GraphQLObjectType<any, GQLContext>({
     dimensions: {
       type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(TemplateDimension))),
       description: 'The dimensions that are part of this template',
-      resolve: async ({id: templateId}, _args, {dataLoader}) => {
+      resolve: async ({id: templateId}, _args: unknown, {dataLoader}) => {
         const dimensions = await dataLoader.get('templateDimensionsByTemplateId').load(templateId)
-        const activeDimensions = dimensions.filter(({removedAt}) => !removedAt)
+        const activeDimensions = dimensions.filter(({removedAt}: {removedAt: Date}) => !removedAt)
```

Comment:
this one seems odd, should already be `Date | undefined`. were you getting a noImplicityAny here too?

Reply:
I was! It knows the type but still shouts at me 🤷‍♂️  I've changed `Date` to `Date | undefined`

![removed-at](https://user-images.githubusercontent.com/39854876/143009506-a97df83a-d719-4f72-8b84-eb486f22d1d5.png)